src/Network/HTTP/Download/Verified.hs

```diff
@@ -322,3 +324,21 @@ verifiedDownload DownloadRequest{..} destpath progressSink = do
                   *> maybe (pure ()) (assertLengthSink drRequest) drLengthCheck
                   *> ZipSink sink
                   *> ZipSink (progressSink mcontentLength))
+
+
+
+-- | Like 'UnliftIO.Temporary.withTempFile', but the file is created with
+--   default file permissions, instead of read/write access only for the owner.
+withTempFileWithDefaultPermissions
+             :: MonadUnliftIO m
+             => FilePath -- ^ Temp dir to create the file in.
+             -> String   -- ^ File name template. See 'openTempFile'.
+             -> (FilePath -> Handle -> m a) -- ^ Callback that can use the file.
+             -> m a
+withTempFileWithDefaultPermissions tmpDir template action =
+  bracket
+    (liftIO (openTempFileWithDefaultPermissions tmpDir template))
+    (\(name, handle') -> liftIO (hClose handle' >> ignoringIOErrors (removeFile name)))
+    (uncurry action)
+  where
+    ignoringIOErrors = liftM (const ()) . tryIO -- yes, it's just void, but for pre-AMP GHCs
```

Comment:
Using `liftM` here triggers a CI failure:

https://dev.azure.com/commercialhaskell/http-download/_build/results?buildId=2622&view=logs&jobId=4556e1b3-9539-56cd-971d-ac2ca862cf24&j=4556e1b3-9539-56cd-971d-ac2ca862cf24&t=5e9614ff-3cd8-56ab-009b-b533691987fd

Can you switch it to `fmap`? Actually, given that we don't support pre-AMP GHCs, using `void . tryIO` is probably just fine.

Reply:
Changed to use `void`